components/FavPokemon.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { useEffect, useState } from 'react';
+import { StyleSheet, Text, View, Image, Button } from 'react-native';
+
+const FavPokemon = ({ navigation }) => {
```

Comment:
nit -> I don't mind, but just wanted to let you know that some people prefer using `function FavPokemon` when declaring components, not `const`s, or even force using only one version for consistency.